ui/src/app/utils/index.tsx

```diff
@@ -533,6 +536,19 @@ export function validateInputForMySQL(searchString: string): Array<string> {
     // unclosed quote
     inputErrors.add('There is an unclosed " in the search string');
   }
+  // check each endsWith term - terms that start with *
+  searchString.split(' ').forEach((word) => {
+    if (word.substring(1).length < searchTrigger && word.startsWith('*')) {
```

Comment:
This works for ends with terms, shouldn't we also be checking starts with, not present and quoted terms?
The following search strings will call the api when it seems like we should show the minimum length error:

- `r*`
- `-r`
- `"r"`
- `""`

Reply:
No, because for endsWith we use each term separately. for those words that does not start with *, all the terms are concatenated and when we do a modifedTerm, the term is matched against fulltext index such that any words that are one/2 letters are ignored.
Again, yes, I think we should check of `every word without special char` to adhere to min-length, such that all the above examples you cited would be invalid.

Reply:
done!